distribution/lib/Standard/Test/0.0.0-dev/src/Main.enso

```diff
@@ -234,7 +234,7 @@ Any.should_fail_with self matcher frames_to_skip=0 =
 Error.should_fail_with : Any -> Integer -> Assertion
 Error.should_fail_with self matcher frames_to_skip=0 =
     caught = self.catch
-    if caught.is_a matcher then Nothing else
+    if (caught == matcher) || caught.is_a matcher then Nothing else
```

Comment:
shouldn't this be folded into `is_a` given the matcher is a `type`.

Reply:
yeah, correct, this is leftover from a previous attempt